authorizer-app-backend/src/main/java/org/radarbase/authorizer/resources/RestSourceUserResource.kt

```diff
@@ -74,7 +74,7 @@ class RestSourceUserResource(
                 .map { it.id }
         }
 
-        if (projects.isEmpty()) return RestSourceUsers(emptyList())
+        if (projects.isEmpty()) return RestSourceUsers(emptyList(), Page())
```

Comment:
for completeness, you can set
```suggestion
        if (projects.isEmpty()) return RestSourceUsers(emptyList(), Page(totalElements = 0))
```